platforms/ios/framework/src/TGMapView.mm

```diff
@@ -38,6 +38,15 @@ inline CLLocationDirection convertRotationRadiansToBearingDegrees(float rotation
     return TGDegreesFromRadians(-rotation);
 }
 
+/**
+ Map region change states
+ */
+typedef NS_ENUM(NSInteger, TGMapRegionChangeStates) {
+    IDLE = 0,
```

Comment:
Enum values should use our "TG" prefix because unlike Java everything is in the global namespace :\

Conventionally they would be something like `TGMapRegionIdle`.

Reply:
Yeah I always forget this!!!!! Will update.

Reply:
Unless you mark it as static, then its scoped to the file, but lets not do that :)